mapzen-places-api/src/main/java/com/mapzen/places/api/ui/PlaceAutocompleteView.java

```diff
@@ -0,0 +1,99 @@
+package com.mapzen.places.api.ui;
+
+import com.mapzen.places.api.R;
+import com.mapzen.places.api.internal.PlaceAutocompleteActivity;
+
+import android.app.Activity;
+import android.app.Fragment;
+import android.content.Context;
+import android.content.Intent;
+import android.util.AttributeSet;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.widget.LinearLayout;
+import android.widget.TextView;
+
+/**
+ * Compound view which displays search icon, autocomplete text view and clear button.
+ */
+public class PlaceAutocompleteView extends LinearLayout {
```

Comment:
It can be moved to the internal package, we still need the public scope because it's accessed by the `PlaceAutocompleteFragment`.

I think we should keep the class in this module because the `pelias-android-sdk` will not know about the Place activities we launch. After that reworking is done, we could update this class to inherit from the `PeliasSearchView`, remove the redundant assets from this module, and override what happens when the user clicks the input text.